src/components/Header/index.tsx

```diff
@@ -284,56 +276,16 @@ export default function Header() {
       </Modal>
       <HeaderRow>
         <Title to="/">
-          <UbeIcon>
-            <img
-              width={isMobile ? '100px' : '215px'}
-              height={isMobile ? '86px' : '80px'}
-              src={Icon}
-              alt="Revo.Finance"
-            />
-          </UbeIcon>
+          <img height={isMobile ? '48px' : '60px'} src={Icon} alt="Revo.Finance" />
         </Title>
-        <HeaderLinks>
-          <StyledNavLink id={`swap-nav-link`} to={'/swap'}>
-            {'Zap In'}
-          </StyledNavLink>
-          <StyledNavLink
-            id={`pool-nav-link`}
-            to={'/pool'}
-            isActive={(match, { pathname }) =>
-              Boolean(match) ||
-              pathname.startsWith('/add') ||
-              pathname.startsWith('/remove') ||
-              pathname.startsWith('/create') ||
-              pathname.startsWith('/find')
-            }
-          >
-            {t('pool')}
-          </StyledNavLink>
-          <StyledNavLink id="compound-nav-link" to="/compound">
-            {t('compound')}
-          </StyledNavLink>
-          {/* <StyledNavLink id="farm-nav-link" to="/farm">
-            {t('farm')}
-          </StyledNavLink> */}
-          {/* <StyledNavLink id={`bridge-nav-link`} to={'/bridge'}>
-            {t('bridge')}
-          </StyledNavLink>
-          <StyledNavLink id={`stake-nav-link`} to={'/stake'}>
-            {t('stake')}
-          </StyledNavLink>
-          <StyledExternalLink id={`stake-nav-link`} href={'https://info.ubeswap.org'}>
-            {t('charts')} <span style={{ fontSize: '11px' }}>↗</span>
-          </StyledExternalLink> */}
-        </HeaderLinks>
       </HeaderRow>
       <HeaderControls>
         <HeaderElement>
-          <HideSmall>
+          {/* <HideSmall>
             {chainId && NETWORK_LABELS[chainId] && (
               <NetworkCard title={NETWORK_LABELS[chainId]}>{NETWORK_LABELS[chainId]}</NetworkCard>
             )}
-          </HideSmall>
+          </HideSmall> */}
```

Comment:
Is this the thing that tells you whether you're on Alfajores or Mainnet? Seems useful to have it if so

Reply:
totally fine with punting this and putting it in an issue though, probably post-mvp since it's not 100% critical (the only people who mess with Alfajores should probably know to check the network before depositing huge amounts)

Reply:
yeah good question - actually it doesn't tell you about the environment, it'd be helpful if it did though. maybe we should have something like this. but this `NetworkCard`,  all that is displayed is "Celo" here (just checked on Ubeswap alfajores and mainnet as well) i removed this because we're not supporting other chains for now and it was just taking up space

Reply:
actually this is probably a bug, i'll try to fix it